src/components/SearchForm/SearchForm.tsx

```diff
@@ -34,7 +34,7 @@ const SearchForm = () => {
           <SearchBar
             id="mainContent"
             action={`${BASE_URL}/search`}
-            method="get"
+            method="post"
```

Comment:
why is this method changing?

Reply:
I did it to have parity with the advanced search form (it's using a post request in the DFE app) but it might make sense to change them both to GET since it'll include the query string in the request.

Reply:
Replaced both with a GET, think that makes more sense for search requests.

Reply:
Does that make it still work when js is turned off? I thought POST was the more appropriate one for the static (basic?) html form?

Reply:
After taking a closer look, I think I understand why advanced search was using a post on the old app. Rather than using a query string to process the search, the request body parameters are used to build the redirect query string on the server side. I will change to a post (only on advanced search) for now and add this functionality in my advanced search utils PR)